guides/plugins/plugins/storefront/using-the-datepicker-plugin.md

```diff
@@ -0,0 +1,190 @@
+# Using the datepicker plugin
+
+## Overview
+
+To provide an input field for date and time values, you can use the datepicker plugin. This guide shows you how to use it.
+
+The datepicker plugin uses the `flatpickr` implementation under the hood. So, check out the `flatpickr` documentation,
+if you need more information about the date picker configuration itself.
+
+<!-- markdown-link-check-disable-next-line -->
+{% embed url="https://flatpickr.js.org" caption="" %}
+
+## Prerequisites
+
+You won't learn how to create a plugin in this guide, head over to our Plugin base guide to create
+your first plugin:
+
+{% page-ref page="../plugin-base-guide.md" %}
+
+You should also know how to customize templates:
+
+{% page-ref page="./customize-templates.md" %}
+
+## Setup a datepicker input field
+
+To apply the datepicker functionality we have to add a DOM element like an input field in a template.
+To keep this example simple for now we just override the `base_main_inner` of the `storefront/page/content/index.html.twig`.
+
+{% code title="<plugin root>/src/Resources/views/storefront/page/content/index.html.twig" %}
+```text
```

Comment:
Dunno, if this makes a difference, but we use twig at other places, so maybe we should stick to it

``````suggestion
```twig
``````